ui/ozone/platform/wayland/wayland_pointer.h

```diff
@@ -25,6 +25,7 @@ class WaylandPointer {
   }
 
   int GetFlagsWithKeyboardModifiers();
+  void ResetFlags();
```

Comment:
Can you add a comment why it can be called?

Reply:
done